wdl-grammar/tests/parsing/unbound-decl-in-output/source.errors

```diff
@@ -0,0 +1,8 @@
+  × expected `=`, but found `}`
+    ╭─[tests/parsing/unbound-decl-in-output/source.wdl:9:5]
+  8 │         String x
+  9 │     }
+    ·     ┬
+    ·     ╰── unexpected `}`
+ 10 │ }
+    ╰────
```

Comment:
We should do a follow-up PR that allows additional help text to better clarify that declarations in output sections must be assigned a value.